contract/test/test-bundle-source.js

```diff
@@ -8,7 +8,8 @@ import { test } from './prepare-test-env-ava.js';
 
 import { createRequire } from 'module';
 import bundleSource from '@endo/bundle-source';
-import { E, passStyleOf } from '@endo/far';
+import { E } from '@endo/eventual-send';
+import { passStyleOf } from '@endo/far';
```

Comment:
pls revert

Reply:
This was added since anytime E is imported from `@endo/far`, instead of `@endo/eventual-send`, typescript will complain that  **This expression is not callable.  Type 'never' has no call signatures**
<img width="534" alt="image" src="https://github.com/Agoric/dapp-offer-up/assets/11021913/83a93614-b770-4861-b7f5-dc3c521d71a4">

Reply:
Sounds like a bug in `@endo/far`'s type exports. Hopefully fixed in subsequent releases.

Either way, eventual-send is the source: https://github.com/endojs/endo/blob/2179108cb9247e9499c1f319de907d7cd365f314/packages/far/src/index.js#L1

`@endo/far` is just for convience. https://github.com/endojs/endo/blob/master/packages/far/README.md#L6

Reply:
as noted [above](#discussion_r1438468079), `@endo/far` is the norm / convention. It started a couple years ago. I'm not in a position to approve a migration away from it on my own.

 - https://github.com/Agoric/agoric-sdk/pull/4144
 - https://github.com/Agoric/agoric-sdk/pull/7261

Reply:
Tracking this here https://github.com/Agoric/dapp-offer-up/issues/31 to keep this PR moving